doc/release/0.5.0-notes.rst

```diff
@@ -22,19 +109,92 @@ Backwards incompatible changes
 Bugs Fixed
 ==========
 
+``demo/image_blender.py`` was updated to support the new api of Pillow 3.x
+
+A bug related to size of assumed ``size_t`` on some platforms/compilers
+(e.g. Windows with mingw64) was fixed.
+
+Fix to memory leak in ``(i)dwt_axis``
+
+Fix to a performance regression in ``idwt`` and ``iswt`` that was introduced
+in v0.4.0.
+
+Fixed a bug in ``dwtn`` and ``idwtn`` for data with complex dtype when
+``axes != None``.
 
 Other changes
 =============
 
+The minimum supported numpy version has been increased to 1.9.1.
+
+Test coverage (including for the Cython and C code) via
+`Codecov <https://codecov.io/>`_ was added and the overall test coverage has
+been improved.
+
+A substantial overhaul of the C extension code has been performed.  Custom
+templating is no longer used.  The intention is to make this code easier to
+maintain and expand in the future.
+
+The Cython code has been split out into a multiple files to hopefully make
+relevant portions of the wrappers easier to find for future developers.
+
+``setup.py`` now relies on setuptools in all cases (rather than distutils).
 
 Authors
 =======
 
+* asnt +
```

Comment:
Hi, my real name is Alexandre Saint. Feel free to put it there. Thanks for your effort.